### PR TITLE
Update openOMP to compile for Linux

### PR DESCRIPTION
**Pull request by JoeOster**
# Existing Sample Changes
## Description

This is a change to original sample. Originally it was written for Mac only, I am extending the sample.json so that it will work for Linux as well.

Also fixing a template

Fixes Issue# 

## External Dependencies

N/A

## Type of change

Please delete options that are not relevant. Add a 'X' to the one that is applicable. 

- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] When compiling the compiler flag "-Wall -Wformat-security -Werror=format-security" was used

### CODE REVIEW

**Comment by JoeOster** [comment]
We need ci to run to verify the sample.json works as expected

File: DirectProgramming/C++/CombinationalLogic/MandelbrotOMP/sample.json [hunk 1]
```diff
@@ -11,6 +11,10 @@
         "darwin": [ 
             { "id": "standard", "steps": [ "make", "make run", "make clean" ] },
             { "id": "perf_num", "env": [ "export perf_num=1" ], "steps": [ "make", "make run", "make clean" ] }
-        ]
+        ],
+	    "linux": [ 
+            { "id": "standard", "steps": [ "make", "make run", "make clean" ] },
+            { "id": "perf_num", "env": [ "export perf_num=1" ], "steps": [ "make", "make run", "make clean" ] }
+        ],
```
Comment (mdbtucker) [inline DirectProgramming/C++/CombinationalLogic/MandelbrotOMP/sample.json:18]: Probably don't want a comma here

Comment (JoeOster) [reply DirectProgramming/C++/CombinationalLogic/MandelbrotOMP/sample.json]: Fixed

File: .github/ISSUE_TEMPLATE/new_sample.md [hunk 2]
```diff
@@ -21,9 +21,10 @@ Please supply what [Domain](https://github.com/oneapi-src/oneAPI-samples/wiki/Re
 Include a short Description of what the sample while Demonstrate and why it is important to the ecosystem.
 
 ## Dependencies
-Please list any dependencies, libraries and images, etc and urls to the download locations
+Please list any dependencies, libraries and images, etc and urls to the download locations using the following [form](https://teams.microsoft.com/l/file/576476E2-256A-4AF9-A1A9-9619A3DB9EB2?tenantId=46c98d88-e344-4ed4-8496-4ed7712e255d&fileType=pptx&objectUrl=https%3A%2F%2Fintel.sharepoint.com%2Fsites%2FOneAPICodeSamplesCommittee%2FShared%20Documents%2FGeneral%2FNewSample_Approval_Template_v0.7.pptx&baseUrl=https%3A%2F%2Fintel.sharepoint.com%2Fsites%2FOneAPICodeSamplesCommittee&serviceName=teams&threadId=19:d29358239f3343f2b26f0c13a47a478d@thread.skype&groupId=1bc67a43-c3a0-4246-98e2-21f851672b78)
```
Comment (mdbtucker) [inline .github/ISSUE_TEMPLATE/new_sample.md:24]: This link opens MS Teams, is that the intent?  Is this an appropriate link to include in a public repo?

Comment (JoeOster) [reply .github/ISSUE_TEMPLATE/new_sample.md]: removed

**Review by mdbtucker — APPROVED** [review]

**Pull request merged**